lib/list.g

```diff
@@ -875,3 +875,107 @@ DeclareSynonym( "IntersectBlist", INTER_BLIST );
 ##  <#/GAPDoc>
 ##
 DeclareSynonym( "SubtractBlist", SUBTR_BLIST );
+
+#############################################################################
+##
+#F  FlipBlist( <blist> )
+##
+##  <#GAPDoc Label="FlipBlist">
+##  <ManSection>
+##  <Func Name="FlipBlist" Arg='blist'/>
+##
+##  <Description>
+##  Changes every entry in <A>blist</A> that equals <K>true</K> to <K>false</K>
+##  and vice versa. If <C>blist1</C> and <C>blist2</C> are boolean list with
```

Comment:
@wilfwilson you can also make code suggestions like so, which then the PR submitter can apply directly, thus saving some time (and often it also makes it clearer what is being suggested).
```suggestion
##  and vice versa. If <C>blist1</C> and <C>blist2</C> are boolean lists with
```